lib/services/local/events_test.go

```diff
@@ -0,0 +1,125 @@
+/*
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
```

Comment:
does the copyright header need to be updated for backports?

Reply:
yeah it should be, I guess the bot doesn't do that - will update and double check the others

Reply:
from grep'ing for the AGPL license, looks like lots of backports have backported it lol.